x/rollup/keeper/msg_server.go

```diff
@@ -227,3 +229,24 @@ func (k *Keeper) SetL1BlockHistory(ctx context.Context, info *derive.L1BlockInfo
 func evmToCosmos(addr common.Address) sdk.AccAddress {
 	return addr.Bytes()
 }
+
+// TODO: This is a temporary change while the rollup module refactor is being done. Change when possible
+func ValidateBasic(m *rollupv1.InitiateWithdrawalRequest) error {
```

Comment:
It's probably worth renaming this `validateBasic` for now while the func is declared here since it doesn't need to be exported and to avoid confusion with the message interface. It will be renamed to `ValidateBasic` with the message as the method receiver after the rollup refactor is done.